doc/source/cluster/kubernetes/user-guides/kuberay-gcs-ft.ipynb

```diff
@@ -0,0 +1,900 @@
+    "    print(ray.get(counter.increment.remote()))\n",
+    "    ```\n",
+    "\n",
+    "### Step 5: Create a detached actor\n"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": 7,
+   "id": "978b889d-66af-408a-bd24-c36273edadb9",
+   "metadata": {
+    "editable": true,
+    "slideshow": {
+     "slide_type": ""
+    },
+    "tags": [
+     "nbval-ignore-output"
+    ]
+   },
+   "outputs": [
+    {
+     "name": "stdout",
+     "output_type": "stream",
+     "text": [
+      "2025-04-14 18:53:36,998\tINFO worker.py:1514 -- Using address 127.0.0.1:6379 set in the environment variable RAY_ADDRESS\n",
+      "2025-04-14 18:53:37,001\tINFO worker.py:1654 -- Connecting to existing Ray cluster at address: 10.244.0.8:6379...\n",
+      "2025-04-14 18:53:37,158\tINFO worker.py:1832 -- Connected to Ray cluster. View the dashboard at \u001b[1m\u001b[32m10.244.0.8:8265 \u001b[39m\u001b[22m\n",
+      "2025-04-14 18:53:40,273\tINFO worker.py:1514 -- Using address 127.0.0.1:6379 set in the environment variable RAY_ADDRESS\n",
+      "2025-04-14 18:53:40,276\tINFO worker.py:1654 -- Connecting to existing Ray cluster at address: 10.244.0.8:6379...\n",
+      "2025-04-14 18:53:40,607\tINFO worker.py:1832 -- Connected to Ray cluster. View the dashboard at \u001b[1m\u001b[32m10.244.0.8:8265 \u001b[39m\u001b[22m\n",
+      "1\n"
+     ]
+    }
+   ],
+   "source": [
+    "# Step 5.1: Create a detached actor with the name, `counter_actor`.\n",
+    "export HEAD_POD=$(kubectl get pods --selector=ray.io/node-type=head -o custom-columns=POD:metadata.name --no-headers)\n",
+    "kubectl exec -it $HEAD_POD -- python3 /home/ray/samples/detached_actor.py\n",
+    "# Step 5.2: Increment the counter.\n",
+    "kubectl exec -it $HEAD_POD -- python3 /home/ray/samples/increment_counter.py"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "id": "6a1817cd-4380-4c28-b867-a3f55cc20910",
+   "metadata": {
+    "editable": true,
+    "slideshow": {
+     "slide_type": ""
+    },
+    "tags": []
+   },
+   "source": [
+    "\n",
+    "(kuberay-external-storage-namespace-example)=\n",
+    "### Step 6: Check the data in Redis\n"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "id": "0454a9bb-4908-4c69-95c9-b2d1c2eb42cc",
+   "metadata": {
+    "editable": true,
+    "slideshow": {
+     "slide_type": ""
+    },
+    "tags": [
+     "nbval-ignore-output",
+     "remove-output"
+    ]
+   },
+   "outputs": [],
+   "source": [
+    "# Step 6.1: Check the RayCluster's UID.\n",
+    "kubectl get rayclusters.ray.io raycluster-external-redis -o=jsonpath='{.metadata.uid}'\n",
+    "# [Example output]: 864b004c-6305-42e3-ac46-adfa8eb6f752"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "id": "d8e5d436-f9a4-44ef-86e4-635431caa34e",
+   "metadata": {
+    "editable": true,
+    "slideshow": {
+     "slide_type": ""
+    },
+    "tags": [
+     "nbval-ignore-output",
+     "remove-output"
+    ]
```

Comment:
```suggestion
    "tags": [
     "nbval-ignore-output"
    ]
```
Ignore `nbval` for this cell but don't remove output for this cell. Also removes the `# [Example output]` above. ditto for step 6.2, 6.3, 6.4, 6.5, 7.1, 9.4, 9.5.